.nengobones.yml

```diff
@@ -103,6 +103,7 @@ setup_py:
   # note: docs/tests_req are intentionally empty to test that the correct
   # requirements are built into the ci scripts
   install_req:
+    - black>=19.3b0
```

Comment:
Turns out that black only supports Python>=3.6, so this would mean nengo-bones will no longer support 3.5. Do we want to make that change? Or just drop this black autoformatting?

Reply:
In either case we'll need to make more changes if we want to support 3.5 (probably using conda to create a separate Python>=3.6 environment in which to run the black commands during the static checks). But it also means in general that no one working in a 3.5 environment will be able to conform to our style guide.

Reply:
Yeah good point ... I think I'm fine with requiring 3.6 in Nengo Bones but having downstream repos continue to support 3.5 (even though devs would need access to Python 3.6).

Reply:
Added a fixup so that things will run on 3.5, but anything related to black is skipped (which should be fine as long as downstream repos have at least some builds running >=3.6)